optimum/exporters/openvino/model_patcher.py

```diff
@@ -39,6 +41,14 @@ def patch_model_with_bettertransformer(model):
             + COLOR_RESET
         )
 
+    if is_transformers_version(">=", "4.38") and _openvino_version < "2024.1.0-14612":
```

Comment:
why not :
```suggestion
    if is_transformers_version(">=", "4.38") and is_optimum_version("<", "2024.1.0-14612"):
```

Reply:
also this concerns only a subset of architectures, so would make sense to only have a warning for those wdyt ?

Reply:
Changed both. Do models based on llama keep model_type llama?

Reply:
[Here](https://github.com/huggingface/transformers/blob/b382a09e28c7e59129246ccdf4b00f2cac4547a4/src/transformers/models/auto/configuration_auto.py#L274) you can find the model's type list